### PR TITLE
Replace "Collections.sort" with "servers.sort"

### PR DESCRIPTION
**Pull request by turncoat54**

### CODE REVIEW

**Comment by turncoat54** [comment]
alluxio-bot, check this please

**Comment by newnius** [comment]
Jenkins, test this please.

**Comment by newnius** [comment]
/usr/src/alluxio/core/server/common/src/main/java/alluxio/Registry.java:23:8: error: Unused import - java.util.Collections.

File: core/server/common/src/main/java/alluxio/Registry.java [hunk 1]
```diff
@@ -114,7 +114,8 @@ public Registry() {}
    */
   public List<T> getServers() {
     List<T> servers = new ArrayList<>(mRegistry.values());
-    Collections.sort(servers, new DependencyComparator());
+    //Collections.sort(servers, new DependencyComparator());
```
Comment (aaudiber) [inline core/server/common/src/main/java/alluxio/Registry.java]: please remove this line

**Comment by apc999** [comment]
Hi @turncoat54 can you remove `//Collections.sort(servers, new DependencyComparator());` ?

**Comment by newnius** [comment]
@apc999 Duplicate https://github.com/Alluxio/alluxio/pull/8647

**Comment by apc999** [comment]
@newnius thanks for the reminder. Since this issue has been fixed in #8647, I will close this PR.

**Pull request closed without merging**